commons-db/src/main/java/com/palantir/nexus/db/pool/CloseTracking.java

```diff
@@ -78,9 +78,8 @@ public synchronized void close() {
 
         public synchronized void check() {
             if (!closed) {
-                String errorMessage = typeName + " never closed!";
-                log.error(errorMessage, createTrace);
-                assert false : errorMessage + "\n" + createTrace.getStackTrace();
+                log.error("{} never closed!", typeName, createTrace);
+                assert false : typeName + " never closed!" + "\n" + createTrace.getStackTrace();
```

Comment:
Do we require products run with assertions enabled?  If not we need to figure out if these lines are valid.